cypress/support/commands.ts

```diff
@@ -28,14 +28,21 @@ Cypress.Commands.add("signup", () => {
         .find("pl-button#submitEmailButton")
         .click({ force: true });
 
-    // Give the app some time to finish animations
-    cy.wait(100);
-
-    cy.get("pl-app")
-        .find("pl-prompt-dialog")
-        .find("pl-input")
-        .find("input[placeholder='Enter Verification Code']")
-        .type(emailToken, { force: true });
+    // Give the app some time to finish animations and email to arrive
+    cy.wait(2000);
+
+    cy.request("http://localhost:1080/email").should((res) => {
+        const latest = res.body.sort((a, b) => (a.time > b.time ? -1 : 1))[0];
+        expect(latest).to.not.be.undefined;
+        const matchCode = latest.text.match(/(\d{6})/);
+        const emailToken = matchCode && matchCode[1];
+        expect(emailToken).to.not.be.null;
+        cy.get("pl-app")
+            .find("pl-prompt-dialog")
+            .find("pl-input")
+            .find("input[placeholder='Enter Verification Code']")
+            .type(emailToken, { force: true });
+    });
```

Comment:
The approach is pretty straighforward: Grab the latest email from the [maildev rest endpoint](https://github.com/maildev/maildev/blob/master/docs/rest.md) and extract the verification code. Not sure if there is a better way to do this with cypress though.